Lib/xml/dom/minidom.py

```diff
@@ -204,6 +221,12 @@ def normalize(self):
         self.childNodes[:] = L
 
     def cloneNode(self, deep):
+        """Create and return a duplicate of this node.
+
+        Args:
+            deep (bool): If True, recursively clone this node's descendants.
+                        If False, clone only this node.
```

Comment:
There is no _"Args:"_ convention for parameter docstrings. Please take a look at how Argument Clinic generates parameters docstrings and follow that convention.

Reply:
FTR, 727af86547291334a16a44ccd6b7c8c9f31f20e0 does _not_ align with Argument Clinic's formatting. Please take another look at how it is done.

Reply:
You mean this way?

```
def cloneNode(self, deep):
        """Create and return a duplicate of this node.

        :param deep: If True, recursively clone this node's descendants.
                     If False, clone only this node.
        """
        return _clone_node(self, deep, self.ownerDocument or self)
```

Reply:
No. Please take a look at how Argument Clinic generates docstrings. You'll find Argument Clinic generated code in the `clinic/` directories in the repo. For example:

https://github.com/python/cpython/blob/8d15058d61681e98579cf3fbd3c6fd13b3df6a72/Modules/_sqlite/clinic/connection.c.h#L233-L248

```sh
$ python3 -c "import sqlite3; print(sqlite3.Connection.blobopen.__doc__)"
Open and return a BLOB object.

  table
    Table name.
  column
    Column name.
  row
    Row index.
  readonly
    Open the BLOB without write permissions.
  name
    Database name.
```

Reply:
Addressed in ecb4a54d051d24f6bb8344d92b07a4c7170c6fa3